packages/@aws-cdk/core/test/test.app.ts

```diff
@@ -301,6 +301,31 @@ export = {
     test.done();
   },
 
+  'version reporting includes @aws-solutions-konstruk libraries'(test: Test) {
```

Comment:
1. Extract this to a separate file `test.runtime-info.ts`
2. No need to actually run the entire app. `collectRuntimeInfo` is exported so you should be able to simply call it directly.